### ISSUE TITLE
can't override private method _activateHandler(e) in polymer_selector.dart

### ISSUE THREAD

**Issue by zoechi** [bwu-dart-graveyard/polymer_elements#63]
polymer-ui-accordion (ErikGrimes/polymer_ui_elements#20) derives from polymer_selector and I need to override the _activateHandler(e) method.
can you please change the method to public.

**Comment by ErikGrimes** [comment]
You can go ahead and change this.  All the code is shared responsibility :)

**Comment by ErikGrimes** [comment]
I've been making everything that was not part of the recognizable public
api private.  I've always found that to be good practice.  I might have to
re-evaluate.  I'm not how much sense private methods make with polymer.  So
much of an element internals are public because of callbacks and mixins.

**Comment by zoechi** [comment]
I started with changing all that seemed reasonable to private at first.
Then there was a problem/bug that private members could not be observable.
Overridden methods can't be private either, then there is not to much left.

In the end I went with leaving all public when porting code to avoid more troubles and to have as less differences as possible with the original code.
I don't like it though. 
I kept this (bad) style because there are enough troubles to cope with (as you mentioned in the PM, many breaking changes, bugs, ...).
I make at least fields that are not published or observable private.

If it's not to much trouble for you I would suggest keeping your best practice.
Changing from private to public later isn't breaking someone's code.
I try to improve more in your direction.

PS: I have no problem to change the code myself.
I assigned it to you because I don't like changing anybody's code silently and
also to initiate this discussion because I saw that we follow a different style.

Maybe it would have been more efficient if you had done it because you know the code but it's not that bad if I deal with the elements I didn't port myself ;-)
